asserts/pool.go

```diff
@@ -314,14 +425,46 @@ func (p *Pool) addUnresolved(unresolved *AtRevision, gnum uint16) error {
 		return nil
 	}
 	uniq := unresolved.Ref.Unique()
-	var u *unresolvedRec
+	var u unresolvedAssertRecord
 	if u = p.unresolved[uniq]; u == nil {
 		u = &unresolvedRec{
 			at: unresolved,
 		}
 		p.unresolved[uniq] = u
 	}
-	u.merge(unresolved, gnum, p.groupings)
+
+	urec := u.(*unresolvedRec)
+	urec.merge(unresolved, gnum, p.groupings)
+	return nil
+}
+
+func (p *Pool) addUnresolvedSeq(unresolved *AtSequence, gnum uint16) error {
+	ok, sequence, err := p.isResolvedSeq(unresolved)
+	if err != nil {
+		return err
+	}
+	if ok {
+		// We assume that either the resolving of
```

Comment:
we never arrive here, I'm also not sure that this case can occur for sequences, because themselves they are never prerequisites ?